sympy/polys/matrices/domainmatrix.py

```diff
@@ -1800,7 +1819,7 @@ def rref(self):
         >>> rref_matrix
         DomainMatrix([[1, 0, 0], [0, 1, 0], [0, 0, 1]], (3, 3), QQ)
         >>> rref_pivots
-        (0, 1, 2)
+        [0, 1, 2]
```

Comment:
There was an inconsistency between different implementations about returning a list or a tuple. Now all low-level methods (DDM, SDM and DomanMatrix) return a list and Matrix still returns a tuple.